front/pages/api/create-new-workspace.ts

```diff
@@ -51,8 +52,20 @@ async function handler(
   }
 
   const workspace = await createWorkspace(session);
+  const userRes = await UserResource.fetchByModelId(user.id);
```

Comment:
Why not going through the `Authenticator` here?

Reply:
The session contains a UserTypeWithWorkspace - because createAndLogMembership then uses the membership resource, we will need to get the UserResource at some point